### ISSUE TITLE
Add support for inferring display options for Money type column with crypto currency data

### ISSUE THREAD

**Issue by silentninja** [mathesar-foundation/mathesar#1220]
## Problem
`currency_info.json` dataset is used for selecting currency codes, currency symbols as display options for the `Mathesar_Money` type. The dataset contains fiat currency details only, we should extend the dataset to include cryptocurrency details too.

## Proposed solution

Crypto currency details should be added to `currency_info.json` dataset. The cryptocurrency data object should follow the same structure as the existing fiat currency data.

## Additional context
- Blocked by #1205

**Comment by Cronus1007** [comment]
@silentninja I would like to work upon this issue.

**Comment by silentninja** [comment]
Sure @Cronus1007, I have assigned the issue to you. Please make sure to start working on it after the blocking PR #1205 is merged into master.

**Comment by Cronus1007** [comment]
@silentninja Sure :+1:

**Comment by aadityasinha-dotcom** [comment]
Is this issue available can I work on this issue?

**Comment by kgodey** [comment]
Sure, go ahead @aadityasinha-dotcom. Thanks for your interest!

**Comment by kgodey** [comment]
@aadityasinha-dotcom I unassigned you since we haven't heard from you in a while.

**Comment by JManan** [comment]
i would like to work on this issue, is it available?

**Comment by kgodey** [comment]
Sorry @JManan, this issue is a draft and needs more details before it can be worked on.

**Comment by kgodey** [comment]
Closing this, too old, requirements are likely out of date. We can create a new issue if we need this functionality in the future.
